config/kovan.json

```diff
@@ -14,6 +14,9 @@
   "usd_native_address": "0x0000000000000000000000000000000000000000",
   "usd_native_address_v3": "0x0000000000000000000000000000000000000000",
   "usd_weth_address": "0x0000000000000000000000000000000000000000",
+  "gysr_native_address": "0x0000000000000000000000000000000000000000",
```

Comment:
Don't need to bother on this PR but we can consider removing the kovan configurations/scripts

Reply:
it's already removed in another branch, i'll do some cleanup/merge after this goes in!